wrench/src/main.rs

```diff
@@ -320,7 +320,7 @@ fn main() {
                                  save_type,
                                  size,
                                  args.is_present("rebuild"),
-                                 args.is_present("subpixel_aa"),
+                                 !args.is_present("disable_subpixel_aa"),
```

Comment:
similarly, we might want to move the inversion `!` later down the pipe in order to be consistent with the other options